### PR TITLE
Restrict visibility of email

### PR DESCRIPTION
**Pull request by Wardormeur**
Frankly backend doesn't send the email back, so it's just a preliminary fix of "how it should work"

### CODE REVIEW

File: web/public/js/controllers/user-profile-controller.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   $scope.editMode = false;
   $scope.publicMode = false;
   $scope.publicChampion = false;
+  $scope.loggedInUserIsMemberOfDojoChampion = false;
```
Comment (DanielBrierton) [inline web/public/js/controllers/user-profile-controller.js]: loggedInUserIsDojoChampion?

Comment (Wardormeur) [inline web/public/js/controllers/user-profile-controller.js]: Nope, you want to check the opposite, if the loggedInUser is a member of the profile's dojo he's viewing

Comment (DanielBrierton) [inline web/public/js/controllers/user-profile-controller.js]: To be exact, is it checking that the user's profile being viewed is a member of a dojo that the currently logged in user is a champion of?

Comment (Wardormeur) [inline web/public/js/controllers/user-profile-controller.js]: Nope, if the profile being viewed is a champion for the dojo that the user seeing is belonging to

**Pull request merged**